submissions/task_html-css-popup/index.html

```diff
@@ -0,0 +1,158 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Hooli-Style Popup</title>
+        <link rel="stylesheet" type="text/css" href="style.css">
+    </head>
+    <body>
+        <header class="header">
+            <nav class="nav">
+                <div class="wrapper">
+                        <nav class="menu header__menu">
+                            <ul class="menu__list">
+                                <li class="menu__item"><a href="#" class="menu__link">Watchlist</a></li>
+                                <li class="menu__item"><a href="#" class="menu__link">Subscribe</a></li>
+                                <li class="menu__item"><a href="#" class="menu__link">Sign In</a></li>
+                                <li class="menu__item">
+                                    <input type="checkbox" id="burgerBtn" class="menu__burger-checkbox">
+                                    <label for="burgerBtn" class="menu__link">
+                                        <span class="menu__burger-icon" tabindex="0"></span>
+                                    </label>
+                                    <div class="popup">
+                                        <ul class="submenu popup__submenu">
+                                            <li class="submenu__item">
+                                                <a class="submenu__link" href="#">
+                                                    <img src="cat-food.png" alt="Mackerel">

```

Comment:
For icons it's better set an empty alt attribute, read more [here](https://accessibility.psu.edu/images/imageshtml/#empty)

Reply:
Fixed. Thx :)